packages/@react-spectrum/tooltip/src/TooltipTrigger.tsx

```diff
@@ -88,7 +89,7 @@ function TooltipTrigger(props: SpectrumTooltipTriggerProps) {
           arrowRef: arrowRef,
           ...tooltipProps
         }}>
-        <Overlay isOpen={state.isOpen} nodeRef={overlayRef}>
+        <Overlay isOpen={state.isOpen} nodeRef={overlayRef} container={UNSTABLE_portalContainer}>
```

Comment:
Was this part of the request? It previously didn't support container. I'm trying to think of the case for this because Tooltip is usually the text label for an icon button.

Reply:
I don't understand the question

Reply:
The `container` prop here is on RSP Overlay which we don't expose so it should be fine (said container gets set as the portal container internally). Users can pass `UNSTABLE_portalContainer` to TooltipTrigger though which falls in line with the intended change